integration-tests/utils/aws/dynamoDB.ts

```diff
@@ -0,0 +1,187 @@
+
+import { AUDIT_REQUEST_DYNAMODB } from '../../constants/awsParameters'
+import {
+  ZendeskFormFieldIDs,
+  ZENDESK_END_USER_EMAIL,
+  ZENDESK_END_USER_NAME
+} from '../../constants/zendeskParameters'
+import { dynamoDBTicketDetails } from '../../constants/dynamoDBTicketDetails'
+import { dynamoDBClient } from './dynamoDBClient'
+
+export const populateDynamoDBWithTestItemDetails = async (ticketID: string) => {
+  const ticketDetails = dynamoDBTicketDetails.ticket
+
+  const populateTableParams = {
+    TableName: AUDIT_REQUEST_DYNAMODB,
+    ReturnValues: 'ALL_OLD',
+    Item: {
+      zendeskId: { S: `${ticketID}` },
+      requestInfo: {
+        M: {
+          zendeskId: { S: `${ticketID}` },
+          dateFrom: {
+            S: `${getFieldValue(
+              ticketDetails,
+              ZendeskFormFieldIDs.PII_FORM_REQUEST_DATE_FIELD_ID
+            )}`
+          },
+          dateTo: {
+            S: `${getFieldValue(
+              ticketDetails,
+              ZendeskFormFieldIDs.PII_FORM_REQUEST_DATE_FIELD_ID
+            )}`
+          },
+          identifierType: {
+            S: `${getFieldValue(
+              ticketDetails,
+              ZendeskFormFieldIDs.PII_FORM_IDENTIFIER_FIELD_ID
+            )}`
+          },
+          recipientEmail: {
+            S: `${getFieldValue(
+              ticketDetails,
+              ZendeskFormFieldIDs.PII_FORM_IDENTIFIER_RECIPIENT_EMAIL
+            )}`
+          },
+          recipientName: {
+            S: `${getFieldValue(
+              ticketDetails,
+              ZendeskFormFieldIDs.PII_FORM_IDENTIFIER_RECIPIENT_NAME
+            )}`
+          },
+          requesterEmail: {
+            S: `${ZENDESK_END_USER_EMAIL}`
+          },
+          requesterName: {
+            S: `${ZENDESK_END_USER_NAME}`
+          },
+          dataPaths: {
+            L: getFieldListValues(
+              ticketDetails,
+              ZendeskFormFieldIDs.PII_FORM_CUSTOM_DATA_PATH_FIELD_ID
+            )
+          },
+          eventIds: {
+            L: getFieldListValues(
+              ticketDetails,
+              ZendeskFormFieldIDs.PII_FORM_EVENT_ID_LIST_FIELD_ID
+            )
+          },
+          sessionIds: {
+            L: getFieldListValues(
+              ticketDetails,
+              ZendeskFormFieldIDs.PII_FORM_SESSION_ID_LIST_FIELD_ID
+            )
+          },
+          journeyIds: {
+            L: getFieldListValues(
+              ticketDetails,
+              ZendeskFormFieldIDs.PII_FORM_JOURNEY_ID_LIST_FIELD_ID
+            )
+          },
+          userIds: {
+            L: getFieldListValues(
+              ticketDetails,
+              ZendeskFormFieldIDs.PII_FORM_USER_ID_LIST_FIELD_ID
+            )
+          },
+          piiTypes: {
+            L: getFieldListValues(
+              ticketDetails,
+              ZendeskFormFieldIDs.PII_FORM_REQUESTED_PII_TYPE_FIELD_ID
+            )
+          }
+        }
+      }
+    }
+  }
+
+  let data = null
```

Comment:
You can just declare a const in the `try` block since this is never reassigned. You can also move the `expect` inside too.